tests/unittest/_torch/modules/moe/test_moe_comm.py

```
@@ -1423,94 +1605,99 @@ def _make_test_params():
+                        f"{comm_type}_zero_tokens",
+                    )
                 )
-                if use_low_precision_combine and not _supports_low_precision_combine(config):
-                    continue
-                param_id = case_id
-                if use_low_precision_combine:
-                    param_id += "_lpcombine"
-                params.append(pytest.param(ep_size, config, id=param_id))
 
-    return params
+            for ep_size, base_config, case_id in boundary_cases:
```

Ruff (0.15.18)

[warning] Loop control variable `case_id` not used within loop body

Rename unused `case_id` to `_case_id`

(B007)